### ISSUE TITLE
Pvspeaker not showing correct available devices on macOS in python

### ISSUE THREAD

**Issue by JohnUiterwyk** [Picovoice/pvspeaker#7]
First, amazing work; really appreciate all examples.  Really enjoying the different libraries. I’m using pvrecorder, cobra, and porcupine and loving how easy it’s been to get up and running.

I’m trying to use pvspeaker , but when I call `PvSpeaker.get_available_devices()`, it shows the same list of devices as pvrecorder, that being mic inputs, and none of the audio output devices.

I’m using  MacOS Sonoma 14.5 on a MacBook Pro 2019 x86, and using the python libs with python 3.10.

I was able to use pvrecorder with no issue.

Any idea what might be behind this? Possible bug?

**Comment by JohnUiterwyk** [comment]
steps to reproduce:
```
$ python3 -m venv .venv
$ source .venv/bin/activate
$ pip install pvspeaker
$ python -c "import pvspeaker; print(pvspeaker.PvSpeaker.get_available_devices())"
['HD Pro Webcam C920', 'MacBook Pro Microphone']
```

**Comment by Igor-Mac-Dev** [comment]
I'm having the same problem in windows with node, it's just showing the microphone

**Comment by Igor-Mac-Dev** [comment]
even after uninstalling pvspeaker

**Comment by albho** [comment]
Hey @JohnUiterwyk and @Igor-Mac-Dev, thanks for reporting. This was indeed a bug, that has since been fixed (940ee4d). The latest libraries ([python 1.0.3](https://pypi.org/project/pvspeaker/) and [node 1.0.1](https://www.npmjs.com/package/@picovoice/pvspeaker-node)) should no longer have this issue.

I'm going to close this issue, feel free to re-open if the issue persists.

**Comment by JohnUiterwyk** [comment]
@albho awesome I can confirm this resolved the issue for me. Thanks and well done!
